FullPageOS/src/modules/rpi-deploy/filesystem/home/pi/updatecli/new_stuff/updatecli.d/updatecli_github_commit.yml

```diff
@@ -0,0 +1,26 @@
+name: Pull latest release on GitHub
+sources:
+  newRelease:
+    kind: githubrelease
+    # name: github version
+    spec:
+      owner: jamtools
+      repository: jamscribe
+      token: "{{ requiredEnv `GITHUB_TOKEN` }}"
```

Comment:
We also need to make sure we place this value into an env var available in the systemd service running this